roles/reproducer/tasks/prepare_networking.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- name: Ensure networks are consistent
+  ansible.builtin.import_tasks: "network_assertions.yml"
```

Comment:
Is there a way to disable the validations?
The ci-framework is a developer / QE tool - I don't think we should block people that need to wire in custom stuff like a pre-existing network etc.

Reply:
done.